modules/mobile/app/controllers/mobile/v0/payment_information_controller.rb

```diff
@@ -32,15 +32,26 @@ def index
       end
 
       def update
+        lh_error_response = nil
+
         payment_information = if lighthouse?
-                                lighthouse_adapter.parse(lighthouse_service.update_payment_info(pay_info))
+                                begin
+                                  lighthouse_adapter.parse(lighthouse_service.update_payment_info(pay_info))
+                                rescue => e
+                                  error = { status: e.status_code, body: e.errors.first }
+                                  lh_error_response = Mobile::V0::Adapters::LighthouseDirectDepositError.parse(error)
+                                end
                               else
                                 evss_proxy.update_payment_information(pay_info)
                               end
 
-        render json: Mobile::V0::PaymentInformationSerializer.new(@current_user.uuid,
-                                                                  payment_information.payment_account,
-                                                                  payment_information.control_information)
+        if lh_error_response
```

Comment:
I'm not sure why Tom Harrison didn't write Lighthouse::DirectDeposit::ErrorResponse as a subclass of base error. If he had, we wouldn't have this condition because it would be automatically handled like nearly all other errors in the vets-api. I haven't looked closely at it, so there may be a good reason he didn't do it that way. 🤷